.github/workflows/kernel_test.yaml

```diff
@@ -59,7 +59,8 @@ jobs:
           else
             echo "❌ Cache MISS - will download dependencies"
           fi
-      - name: Run tests
+      # Run unit tests with JDK 17. These unit tests depend on Spark, and Spark 4.0+ is JDK 17.
```

Comment:
would this comment be better placed besides `java-version: "17"` ?

Reply:
+1 @allisonport-db can you move this